data/css/modules/card/_base.scss

```diff
@@ -38,14 +38,23 @@
       min-width: 96px;
       min-height: 96px;
       border-radius: 50%;
+      color: rgba(white, .6);
       background-color: rgba(#1e1e1e, .8);
```

Comment:
I used the same color as in the video player, but now that I double check that design I see its says it should be black with .6 opacity